src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/CreateTask.groovy

```diff
@@ -16,16 +16,19 @@
 package net.wasdev.wlp.gradle.plugins.tasks
 
 import org.gradle.api.tasks.TaskAction
+import org.gradle.api.GradleException
 
 class CreateTask extends AbstractTask {
 
     @TaskAction
     void create() {
+        
         def params = buildLibertyMap(project);
         if (project.liberty.template != null && project.liberty.template.length() != 0) {
             params.put('template', project.liberty.template)
         }
         executeServerCommand(project, 'create', params)
+        
+        copyConfigFiles();
```

Comment:
Did you look at the maven plugin to see if there were any other places that we call copy files?  We haven't added Install Apps yet, so maybe this is the only other place for now.